src/components/Search.js

```diff
@@ -10,8 +10,28 @@ const Search = props => (
         <div className="form-group search-box">
           <label htmlFor="customerId">Customer Id</label>
           <div className="search-row">
-            <input type="text" id="customerId" className="form-control" placeholder="Customer Id" />
-            <button className="btn btn-primary fn-submit-name">Search IDs</button>
+            <input
+              type="text"
+              id="customerId"
+              className="form-control"
+              placeholder="Customer Id"
+            />
+            <Buttton label="Search IDs" search= {search}/>
+          </div>
+        </div>
+      </div>
+      {/* search by name*/}
+      <div className="col">
```

Comment:
This isn't part of the exercises, but do you notice there's some duplication here - what could you do to improve that?

Reply:
You are right! I have done the changes as you pointed out. I hope I did not make it more complicated!